src/etl/impl/AsyncData.h

```diff
@@ -60,7 +61,12 @@ class AsyncCallData {
                           << " . prefix = " << ripple::strHex(std::string(1, prefix))
                           << " . nextPrefix_ = " << ripple::strHex(std::string(1, nextPrefix_));
 
-        assert(nextPrefix_ > prefix || nextPrefix_ == 0x00);
+        ASSERT(
+            nextPrefix_ > prefix || nextPrefix_ == 0x00,
+            "nextPrefix_ must be greater than prefix. Got: nextPrefix_ = {}, prefix = {} ",
```

Comment:
I think it's better to say "Next prefix must be greater than current prefix." to keep the formatting. 
Also let's remove the last space 🔢